lnd.go

```diff
@@ -134,6 +134,10 @@ func lndMain() error {
 		strings.Title(registeredChains.PrimaryChain().String()),
 		network,
 	)
+	if cfg.Litecoin.SimNet {
```

Comment:
Not needed, no risk running on simnet anyway.

Reply:
I added that here just to inform the user that he can't expect community and LND's team help if encountered problems related to Litecoin and SimNet.

Should I take it out?

Let me know @halseth  please.